test/test-integration/src/test/java/org/wso2am/micro/gw/tests/apiDeploy/APiDeployTestCase.java

```diff
@@ -1,66 +0,0 @@
-/*
- * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
- *
- * WSO2 Inc. licenses this file to you under the Apache License,
- * Version 2.0 (the "License"); you may not use this file except
- * in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied. See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-
-package org.wso2am.micro.gw.tests.apiDeploy;
-
-import com.github.dockerjava.zerodep.shaded.org.apache.hc.core5.http.HttpStatus;
-import io.netty.handler.codec.http.HttpHeaderNames;
-import org.testng.Assert;
-import org.testng.annotations.*;
-import org.wso2am.micro.gw.tests.common.BaseTestCase;
-import org.wso2am.micro.gw.tests.util.*;
-import java.io.File;
-import java.util.HashMap;
-import java.util.Map;
-
-/**
- * Api deploy test cases.
- *
- */
-public class APiDeployTestCase extends BaseTestCase {
```

Comment:
All other test cases depend on this test case right?. Before running all the test cases we need to run this test case because in some cases if this test case fails we can identify from this prior to run other test cases. IMO we need to run this test case or APICTL APIdeploy test case before running all other test cases

Reply:
I'll add this test case back